### DISCUSSION TITLE
Replace certificates

### DISCUSSION THREAD

**Discussion by dkoci** [noobaa/noobaa-core#7107] [General]
Hi.
How to replace certificate for S3 endpoint. The old one is expired. I am running noobaa in OKD cluster.

Tnx

**Comment by guymguym** [comment]
Please take a look at this document - https://github.com/noobaa/noobaa-operator/blob/master/doc/ssl-dns-routing.md
It describes the details of how certificates are deployed to the noobaa endpoints.
I hope this will be clear enough, and let us know if you have any questions.

**Reply by dkoci** [reply]
Thank you. I simply delete S3 route and create a new one with new certificate (re-encrypt TLS termination). I hat to stop the operator because it recreates the route otherwise.
